src/OpenTelemetry/ProcessorPipelineWeight.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+namespace OpenTelemetry;
+
+/// <summary>
+/// Contains definitions for common processor pipeline weights.
+/// </summary>
+internal enum ProcessorPipelineWeight
+{
+    /// <summary>
+    /// Pipeline start. Value: <see cref="int.MinValue"/>.
+    /// </summary>
+    PipelineStart = int.MinValue,
+
+    /// <summary>
+    /// Pipeline enrichment. Value: <c>-10000</c>.
+    /// </summary>
+    /// <remarks>
+    /// Note: Enrichment processors which modify telemetry typically need to run
+    /// before exporters.
+    /// </remarks>
+    PipelineEnrichment = -10_000,
```

Comment:
Why is this needed?

Reply:
I'll remove. Some of this was me exploring how it might work for other cases.